src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -55,9 +72,17 @@ export default function () {
 If you have downloaded the pre-built binary you will find the binary named `xk6-browser`. Using the [example](#example), create a new file (such as `browser_test.js`) in the same directory as the pre-built binary, and paste the example in that file. Now run:
 
 ```
-xk6-browser run browser_test.js
+./xk6-browser run browser_test.js
 ```
 
+<Blockquote mod="note" title="">
+
+Note the `./` prefix, which tells your shell to run the binary located in the current working directory. This is required on macOS and Linux, but not on Windows if you're using the `cmd.exe` shell. If you're using PowerShell, you should specify `.\xk6-browser` instead.
+
+If you installed xk6-browser via a system package, or you've placed the binary in a directory that's part of your `$PATH` environment variable, then you can omit the `./` or `.\` prefix.
```

Comment:
```suggestion
The `./` prefix tells your shell to run the binary located in the current working directory. This is required on macOS and Linux, but not on the Windows `cmd.exe` shell. If you're using PowerShell, specify `.\xk6-browser` instead.

If you installed xk6-browser with a system package, or placed the binary in a directory that's part of your `$PATH` environment variable, you can omit the `./` or `.\` prefixes.
```

Reply:
Small language trimming. I don't think we need to add the word "Note" anymore, since the new `blockquote` component  expresses that already, even for screenreaders.